test/images/pets/redis-installer/on-start.sh

```diff
@@ -25,11 +25,11 @@ PORT=6379
 # Ping everyone but ourself to see if there's a master. Only one pet starts at
 # a time, so if we don't see a master we can assume the position is ours.
 while read -ra LINE; do
-    if [[ "${LINE}" == *"${HOSTNAME}"* ]]; then
-        sed -i -e "s|^bind.*$|bind ${LINE}|" ${CFG}
-    elif [ "$(/opt/redis/redis-cli -h "${LINE}" info | grep role | sed 's,\r$,,')" = "role:master" ]; then
+    if [[ "${LINE[0]}" == *"${HOSTNAME}"* ]]; then
```

Comment:
is this commit identical in behavior?

Reply:
Yes, it is!

Since we use `-a` in read, the result (`LINE`) is an array of the words on the given line.

When `LINE` is an array, `${LINE}` is equivalent to `${LINE[0]}`, and results in the first element in the array.

More info can be found here: https://github.com/koalaman/shellcheck/wiki/SC2128